### PR TITLE
Rien/improvement/tabs

### PR DESCRIPTION
**Pull request by rienheuver**

### CODE REVIEW

File: src/Controllers/WhoopsController.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ public function viewer()
 
     public function index()
     {
-        return response()->json(LwtWhoops::all());
+        return response()->json(LwtWhoops::get());
```
Comment (milo526) [inline src/Controllers/WhoopsController.php:19]: Unnecessary change, what style do we want to keep? Both statements are equal and thus both can be used.

File: src/WhoopsTracker.php [hunk 2]
```diff
@@ -31,7 +30,7 @@ public function report(\Exception $exception)
             ->put('request', $this->requestToArray(request()))
             ->put('trace', $this->exceptionToArray($exception))
             ->put('config', config()->all())
-            ->put('queries', DB::getQueryLog())
+            ->put('environment', $_ENV)
```
Comment (milo526) [inline src/WhoopsTracker.php:33]: Are we no longer interested in the query log?

**Review by milo526 — CHANGES_REQUESTED** [review]

File: src/Controllers/WhoopsController.php (hunk 1 above)
Comment (rienheuver) [reply src/Controllers/WhoopsController.php:19]: Doesn't really matter indeed, was messing around with some options and apparently ended up with this change. `all()` is just a nickname for `get()` though, which I think is kinda unnecessary in the first place. But I'm fine with either option

File: src/WhoopsTracker.php (hunk 2 above)
Comment (rienheuver) [reply src/WhoopsTracker.php:33]: We are, but we can't get to it. The main purpose of LWT is production environments in which the query log is disabled. Enabling the query log in our package adds huge memory usage which is not preferable on production environments. So since we simply can't get to it efficiently, I thought we should let it go.

File: src/Controllers/WhoopsController.php (hunk 1 above)
Comment (milo526) [reply src/Controllers/WhoopsController.php:19]: Let's keep the get in this project, when editing files, changes from all to get can be included

**Review by milo526 — APPROVED** [review]
